### PR TITLE
added ports for postgres network policy

### PR DESCRIPTION
**Pull request by RobinRojowiec**
## Description

Added network policy for Postgres database.

## Chart

Select the chart that you are modifying:
- [x] core
- [ ] dotnet-template
- [ ] pact-broker
- [ ] azure-functions

## Checklist
- [x] Description provided
- [x] Linked issue #41 
- [x] Chart version bumped
- [x] README.md updated with any new values or changes
- [x] Updated template tests in `${CHART}/tests/*.py`

### CODE REVIEW

**Review by RobinRojowiec — COMMENTED** [review]
default postgres policy and set to false

**Review by Imroath — CHANGES_REQUESTED** [review]

File: charts/core/Chart.yaml [hunk 1]
```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 name: charts-core
 description: A Helm chart for Kubernetes
 type: application
-version: 1.1.1
+version: 1.1.2
```
[the remark this answers is not in the document]
Comment (RobinRojowiec) [reply charts/core/Chart.yaml]: Sure, will change that.

File: charts/dotnet-core/Chart.yaml [hunk 2]
```diff
@@ -5,6 +5,6 @@ name: charts-dotnet-core
 type: application
 dependencies:
 - name: charts-core
-  version: 1.1.1
+  version: 1.1.2
```
[the remark this answers is not in the document]
Comment (RobinRojowiec) [reply charts/dotnet-core/Chart.yaml]: Will revert it.

File: charts/dotnet-core/Chart.yaml [hunk 3]
```diff
@@ -5,6 +5,6 @@ name: charts-dotnet-core
...
   repository: "https://ecovadiscode.github.io/charts/"
 version: 2.1.1
```
[the remark this answers is not in the document]
Comment (RobinRojowiec) [reply charts/dotnet-core/Chart.yaml]: Will revert it with the one above and create a separate PR.

**Review by RobinRojowiec — COMMENTED** [review]
set versions to previous ones, prepare separate PR.

**Review by RobinRojowiec — COMMENTED** [review]
resolved

File: charts/core/values.yaml [hunk 4]
```diff
@@ -57,6 +57,7 @@ global:
   lnmElasticNetworkPolicyEnabled: false
   allowAllInternalClusterTrafficNetworkPolicy: false
   sqlNetworkPolicyEnabled: true
+  postgresNetworkPolicyEnabled: true
```
[the remark this answers is not in the document]
Comment (RobinRojowiec) [reply charts/core/values.yaml]: Done.

**Review by RobinRojowiec — COMMENTED** [review]
resolved

**Review by RobinRojowiec — COMMENTED** [review]
resolved

**Review by Imroath — APPROVED** [review]
LGTM

**Pull request merged**